include/libp2p/multi/multicodec_type.hpp

```diff
@@ -30,13 +30,15 @@ namespace libp2p::multi {
       SHA3_384 = 0x15,
       SHA3_256 = 0x16,
       SHA3_224 = 0x17,
-      DAG_PB = 0x70
+      RAW = 0x55,
+      DAG_PB = 0x70,
+      DAG_CBOR = 0x71
```

Comment:
```suggestion
      DAG_CBOR = 0x71,
```
its mostly a matter of taste, but I suggest to put a comma after the last enum item